scene/resources/convex_polygon_shape_2d.cpp

```diff
@@ -82,7 +82,7 @@ void ConvexPolygonShape2D::draw(const RID &p_to_rid, const Color &p_color) {
 	if (is_collision_outline_enabled()) {
 		RenderingServer::get_singleton()->canvas_item_add_polyline(p_to_rid, points, col);
 		// Draw the last segment as it's not drawn by `canvas_item_add_polyline()`.
-		RenderingServer::get_singleton()->canvas_item_add_line(p_to_rid, points[points.size() - 1], points[0], p_color, 1.0, true);
+		RenderingServer::get_singleton()->canvas_item_add_line(p_to_rid, points[points.size() - 1], points[0], p_color, -1.0, true);
```

Comment:
Is that correct: antialiased thin line?

Reply:
Given the polyline right above is drawn with no antialiasing (and the default width) I think we can just make this last segment be drawn in the same manner. So removing the last two arguments (`-1.0, true`) in here should be fine.

Reply:
Actually the polyline above isn't drawn as a thin line (polyline doesn't support it) so that last segment should be drawn as quad (still no antialiasing). So `1.0` should be added back. Sorry for misleading!